src/date-picker/date-picker.scss

```diff
@@ -111,6 +112,35 @@
     @include iui-focus($color: var(--iui-color-foreground-accessory), $offset: -3px);
   }
 
+  &.iui-range-start {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    cursor: pointer;
+  }
+
+  &.iui-range-end {
+    border-top-left-radius: 0;
+    border-bottom-left-radius: 0;
+    cursor: pointer;
+  }
+
+  &.iui-range {
+    border-radius: 0;
+    cursor: pointer;
+    @include themed {
+      background-color: rgba(t(iui-color-foreground-primary-rgb), t(iui-opacity-5));
+      color: t(iui-text-color);
+    }
+  }
+
+  &.iui-range:hover {
```

Comment:
Should the same `font-weight` change and a new `background-color`change be added to `iui-range-start` and `iui-range-end`?

Reply:
hmm.. Maybe we need something, as this is clickable now.
@FlyersPh9 any suggestions on the colors on active days on hover?

Also, I see this focus issue between range and range-end days..
![image](https://user-images.githubusercontent.com/81580355/159299133-c7861029-7e3e-424e-923d-43c1f14ef6d4.png)

Reply:
The focus outline next to start / end of range I think might be ok.  The outline is still visible on 3 of the 4 sides.

Reply:
@FlyersPh9 and any suggestions about hover on range-start and range-end days?

Reply:
I'm thinking leave it as is.  [Google Flights](https://www.google.com/travel/flights) date range picker doesn't have a hover effect on the start / end dates.